apps/src/applab/applab.js

```diff
@@ -1240,6 +1242,10 @@ Applab.execute = function() {
   divApplab.firstChild.focus();
 
   Applab.running = true;
+  $('#headers').addClass('readonly');
+  $('#codeWorkspace').addClass('readonly');
+  // TODO - only if designing?
```

Comment:
Whoops, the TODO was meant for other reasons and I forgot to remove it. I'll check out the hideDesignMode case as well, though I suspect our component may be smart enough to still know not to show anything in this case.